jamon-parent/jamon/src/main/java/org/wicketstuff/jamon/component/EmptyMarkupContainer.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.wicketstuff.jamon.component;
+
+import org.apache.wicket.markup.html.WebMarkupContainer;
+
+/**
+ * Empty markup container that can be swapped in and out when doing ajax replacements.
+ * 
+ * @author lars
+ *
+ */
+@SuppressWarnings("serial")
+public class EmptyMarkupContainer extends WebMarkupContainer
+{
+
+	/**
+	 * Construct. Will use the given id as {@link #setMarkupId(String) markupId} and set the
+	 * {@link #setOutputMarkupId(boolean) outputMarkupId} flag to <code>true</code>.
+	 * 
+	 * @param id
+	 */
+	public EmptyMarkupContainer(String id)
+	{
+		super(id);
+		setMarkupId(id);
```

Comment:
Is this really needed?
There is a chance that there are two HTML elements with the same id in the DOM.